src/Scope/Scope.php

```diff
@@ -0,0 +1,45 @@
+<?php
+declare(strict_types=1);
+
+namespace LanguageServer\Scope;
+
+use Microsoft\PhpParser\Node\QualifiedName;
+
+/**
+ * Contains information about variables at a point.
```

Comment:
Could you elaborate what "point" means here? How is the scope boundary defined?

Reply:
Scope when evaluating the expression at the node being traversed, if it were an expression.

... perhaps? I'm not sure how to express it clearly.